root.tsup.config.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from 'tsup';
+import { esbuildPluginFilePathExtensions } from 'esbuild-plugin-file-path-extensions';
+import { copy } from 'esbuild-plugin-copy';
+
+export default defineConfig([
```

Comment:
I see us adding `sideEffects: false` but not enabling treeshaking here. How come?